mmdet3d/models/data_preprocessors/data_preprocessor.py

```diff
@@ -27,6 +31,9 @@ class Det3DDataPreprocessor(DetDataPreprocessor):
     - Normalize images in inputs with defined std and mean.
```

Comment:
May need to add more comments to present the background/pros & cons of this modification